graph/src/data/subgraph/mod.rs

```diff
@@ -241,6 +241,8 @@ pub enum SubgraphAssignmentProviderError {
     /// Occurs when a subgraph's GraphQL schema is invalid.
     #[fail(display = "GraphQL schema error: {}", _0)]
     SchemaValidationError(failure::Error),
+    #[fail(display = "subgraph attribute indexes build error: {}", _0)]
```

Comment:
Error messages start with a capital letter. Also, this is a bit cryptic. Could we make it `"Error building index for subgraph \"{}\", entity \"{}\" and attribute \"{}\": {}"`?

And add the subgraph deployment ID, entity name and attribute name to the error, if possible.